pkg/cli/cmd/error.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"go.uber.org/zap"
+
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+)
+
+type Error struct {
+	UserMessage string
+	LogMessage  string
+}
+
+func LogError(err *Error, checkLogMessage string) {
+	if err.LogMessage == "" {
```

Comment:
I don't quite understand this logic. If there is no message for the log, why do we instead print the user message as an error?